src/components/MoviesList/MoviesList.scss

```
@@ -6,7 +6,7 @@
 
   list-style-type: none;
 
-  li {
+  &__item {
     flex-basis: 25.1%;
```

```suggestion
    flex-basis: calc(100% / 3);
    display: flex;
    justify-content: center;
```

You can also add some [CSS breakpoints](https://www.w3schools.com/cssref/css3_pr_mediaquery.asp) where you want 4 and 3 cards to be present in one row. Please add it in the way that starting from 1280px width you will have 4 cards in a row, and below that 3.  [Here's an example of how it looks like in the SCSS](https://css-tricks.com/approaches-media-queries-sass/).